pkg/generator/vars.go

```diff
@@ -9,17 +9,19 @@ import (
 
 	log "github.com/sirupsen/logrus"
 
-	"github.com/alexandrst88/terraform-variables-generator/pkg/utils"
+	"github.com/marceloalmeida/terraform-variables-generator/pkg/utils"
 )
 
 var replacer *strings.Replacer
 var varPrefix = "var."
 
-var varTemplate = template.Must(template.New("var_file").Parse(`{{range .}}
-variable "{{ . }}" {
-  description = ""
-}
-{{end}}`))
+var varTemplate = template.Must(template.New("var_file").Funcs(template.FuncMap{"sub": sub}).Parse(`{{- $length := len . -}}
```

Comment:
what's the output will be?

Reply:
I've also updated the `README.md`, the output will look like the following snippet
```
variable "ami" {}

variable "instance_type" {}

variable "cidr" {}

variable "enable_dns_hostnames" {}

variable "enable_dns_support" {}

variable "name" {}
```